crates/astria-cli/src/sequencer/sign.rs

```diff
@@ -0,0 +1,99 @@
+use std::{
+    io::Write,
+    path::{
+        Path,
+        PathBuf,
+    },
+};
+
+use astria_core::{
+    protocol::transaction::v1::TransactionBody,
+    Protobuf,
+};
+use clap_stdin::FileOrStdin;
+use color_eyre::eyre::{
+    self,
+    WrapErr as _,
+};
+
+use crate::utils::signing_key_from_private_key;
+
+#[derive(clap::Args, Debug)]
+pub(super) struct Command {
+    /// The private key of account being sent from
+    #[arg(long, env = "SEQUENCER_PRIVATE_KEY")]
+    // // TODO: https://github.com/astriaorg/astria/issues/594
+    // // Don't use a plain text private, prefer wrapper like from
+    // // the secrecy crate with specialized `Debug` and `Drop` implementations
+    // // that overwrite the key on drop and don't reveal it when printing.
+    private_key: String,
+    /// Target to to write the signed transaction in pbjson format (omit to write to STDOUT).
```

Comment:
nit: remove extra "to"

Reply:
Done in [9ab5f81](https://github.com/astriaorg/astria/pull/1696/commits/9ab5f81be8c6e4895313aa368971925615382246).